### ISSUE TITLE
Doom entirely removes a user.

### ISSUE THREAD

**Issue by PuggyCrew** [TotalFreedom/TotalFreedomMod#1887]
When /doom is used on someone, their entry is removed entirely from the admins.yml file.
I suggest that you add an addition to the mod where /doom's removal process now removes them in saconfig's way.

**Comment by Wild1145** [comment]
The main reason that Doom removes and does not de-activate is because doom is supposed to be a final straw type command that an admin would not usually come back from.

**Comment by Hockeyfan360** [comment]
@Wild1145 It's not usually used like that, though. Half of the time, it's just admins screwing around. Even if it is real, it's probably best to just leave the entry in. If it needs to be deleted manually, that can be done easily.

**Comment by Wild1145** [comment]
@Hockeyfan360 Then said admins need to be suspended and sanctioned. I dont see why we should change the basic functionality of a command because admins act like children.

**Comment by ghost** [comment]
@Hockeyfan360 I see no reason to change the doom command, even if said admins manage to unban themselves and get on, the only pro of having their admin listing still there is that if they were say, Senior Admin, if someone super's them again they will get that rank back. Unless your server is full of headless chooks, that is HIGHELY UNLIKELY TO HAPPEN.

**Comment by LeohPlayzMC1** [comment]
I agree that doom should be kept the way it is - often it's the first thing used when an administrator goes rogue and the whole point of it is to remove and purge an admin to prevent them joining again or anymore harm being done.

If you want a non-official TotalFreedomMod with this feature added then I'm sure you can contact one of the armature TFM developers (There's hundreds out there)
